### PR TITLE
Fix template error

### PR DESCRIPTION
**Pull request by PoulpiFr**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -311,7 +311,7 @@
     },
     "assertion-error": {
       "version": "1.0.2",
-      "resolved": "https://registry.npmjs.org/assertion-error/-/assertion-error-1.0.2.tgz",
+      "resolved": "https://sf-npm.mediarithmics.com/repository/npm-mics/assertion-error/-/assertion-error-1.0.2.tgz",
```
Comment (dduponchel) [inline package-lock.json]: This is not the default registry.

**Comment by dduponchel** [comment]
The type change on the properties is a breaking change, as the existing code (in the examples) doesn't compile anymore:

```


src/MyPluginImpl.ts:350:7 - error TS2322: Type '{ authenticationToken: string | undefined; properties: PropertiesWrapper; }' is not assignable to type 'MyInstanceContext'.
  Types of property 'authenticationToken' are incompatible.
    Type 'string | undefined' is not assignable to type 'string'.
      Type 'undefined' is not assignable to type 'string'.

350       return {
          ~~~~~~~~
351         ...defaultInstanceContext,
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
352         authenticationToken: authenticationToken.value.value
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
353       };
    ~~~~~~~~
```

File: package-lock.json (hunk 1 above)
Comment (PoulpiFr) [reply package-lock.json]: Good catch. I think something got mixed up in my local setup. Fixing it now.

**Comment by PoulpiFr** [comment]
@dduponchel > Well, it is a regression in `v0.4.0`. 
See https://github.com/MEDIARITHMICS/plugins-nodejs-sdk/blob/v0.3.x/src/core/interfaces/mediarithmics/api/common/ValueInterface.ts

How do you want to play it? We release a `v0.5.0` to fix the issue, or do we treat it as a bugfix (`0.4.1` then)?

**Comment by dduponchel** [comment]
Let's roll with a `v0.4.1`.

**Comment by dduponchel** [comment]
released in `v0.4.1`

**Pull request merged**